packages/camera/camera/ios/Classes/CameraPermissionUtils.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Foundation;
+#import <Flutter/Flutter.h>
+
+typedef void (^FLTCameraPermissionRequestCompletionHandler)(FlutterError *);
+
+/// Requests camera access permission.
```

Comment:
Blank comment line after this to make a paragraph please (similar to [Dart style](https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph))